requests_ntlm/requests_ntlm.py

```diff
@@ -33,6 +38,7 @@ def __init__(self, username, password, session=None):
         if self.domain:
             self.domain = self.domain.upper()
         self.password = password
+        self.send_cbt = send_cbt
```

Comment:
I should have added this in the first implementation, it allows an upstream library to not send CBT if they ever not want to. Really just a way to override the behaviour if the code for this doesn't work in an expected way and a temp fix is required to get it going.